File: iOS/SayHi/SayHi/CategoryQuery.swift
//
//  CategoryQuery.swift
//  SayHi
//
//  Created by Klemenz, Oliver on 20.12.16.
//  Copyright © 2016 Klemenz, Oliver. All rights reserved.
//

import Foundation

struct CategoryQuery {
    var favorite: Bool = false    
    var search: Bool = false
    var searchText: String = ""
    var name: String = ""
    var langCode: String = ""
    var primaryLangKey: String = ""
}

File: Android/SayHi/app/src/main/java/de/oklemenz/sayhi/model/CategoryQuery.java
package de.oklemenz.sayhi.model;

import java.io.Serializable;

/**
 * Created by Oliver Klemenz on 22.02.17.
 */

public class CategoryQuery implements Serializable {

    public boolean favorite = false;
    public boolean search = false;
    public String searchText = "";
    public String name = "";
    public String langCode = "";
    public String primaryLangKey = "";
}